### PR TITLE
Added tests config files for amplicons and sispa

### PR DESCRIPTION
**Pull request by saramonzon**
# nf-core/viralrecon pull request

Many thanks for contributing to nf-core/viralrecon!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [x] If necessary, also make a PR on the [nf-core/viralrecon branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/viralrecon)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/viralrecon/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by saramonzon** [comment]
@drpatelh Do you prefer using the direct link for the gff file from ncbi rather than the one we have on the test_datasets for any reason? You can resolve this conflic either way, I don't mind, I just found handy using the one we have in tests!

**Comment by drpatelh** [comment]
I don't think it matters for now @saramonzon. I used that link because the PR to test-datasets hadn't been merged at that point. I think it would be a good idea to upload all of the reference data for the pipeline to Zenodo when we know exactly what we need i.e. kraken2 databases, viral genome etc. Be nice to have everything in a single place.

File: conf/test.config [hunk 1]
```diff
@@ -21,6 +21,7 @@ params {
 
   // Genome references
   host_fasta = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/reference/hg19_chr21.fa'
```
Comment (drpatelh) [inline conf/test.config:23]:
```suggestion

```

Comment (drpatelh) [reply conf/test.config:23]: Don't need this parameter anymore.

File: conf/test_amplicon.config [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  // Limit resources so that this can run on GitHub Actions
+  max_cpus = 2
+  max_memory = 6.GB
+  max_time = 48.h
+
+  // Input data
+  input = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet_test_amplicon.csv'
+
+  // Genome references
+  host_fasta = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/reference/hg19_chr21.fa'
```
Comment (drpatelh) [inline conf/test_amplicon.config:23]:
```suggestion

```

File: conf/test_full.config [hunk 3]
```diff
@@ -16,6 +16,7 @@ params {
 
   // Genome references
   genome = 'hg19'
```
Comment (drpatelh) [inline conf/test_full.config:18]:
```suggestion

```

Comment (drpatelh) [reply conf/test_full.config:18]: Or this one.

File: conf/test_full_amplicon.config [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+params {
+  config_profile_name = 'Full test profile'
+  config_profile_description = 'Full test dataset to check pipeline function'
+
+  // Input data
+  input = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet_full_amplicon.csv'
+
+  // Genome references
+  genome = 'hg19'
```
Comment (drpatelh) [inline conf/test_full_amplicon.config:18]:
```suggestion
  protocol = 'amplicon'
```

File: conf/test_sispa.config [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  // Limit resources so that this can run on GitHub Actions
+  max_cpus = 2
+  max_memory = 6.GB
+  max_time = 48.h
+
+  // Input data
+  input = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet_test_sispa.csv'
+
+  // Genome references
+  host_fasta = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/reference/hg19_chr21.fa'
```
Comment (drpatelh) [inline conf/test_sispa.config:23]:
```suggestion

```

File: conf/test_full.config (hunk 3 above)
Comment (drpatelh) [reply conf/test_full.config:18]: Will need to add links to Kraken databases here when we upload them to Zenodo too.

**Comment by drpatelh** [comment]
Ok. Ill merge these and try and get more tests running with the CI :)

**Pull request merged**